fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/CastVoteFragment.java

```diff
@@ -108,9 +108,9 @@ public View onCreateView(
     // setUp the cast Vote button
     voteButton = mCastVoteFragBinding.castVoteButton;
 
-    // Getting lao
-    Lao lao = mLaoDetailViewModel.getCurrentLao().getValue();
-    if (lao == null) {
+    // Getting laoView
+    LaoView laoView = mLaoDetailViewModel.getCurrentLao().getValue();
```

Comment:
Could we use the `getCurrentLaoValue()` function here ?

Reply:
Indeed, my mistake